Smoke test javaVendorVersion failed with jdk8

Smoke test javaVendorVersion failed with jdk8 as no system property `java.vendor.version`

```
14:35:52  	at net.adoptopenjdk.test.VendorPropertiesTest$AdoptOpenJDKPropertiesChecks.javaVendorVersion(VendorPropertiesTest.java:162)
14:35:52  	at net.adoptopenjdk.test.VendorPropertiesTest.vmPropertiesPointToVendor(VendorPropertiesTest.java:84)
14:35:52  	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
```
https://ci.adoptopenjdk.net/job/build-scripts/job/jobs/job/jdk8u/job/jdk8u-linux-x64-hotspot_SmokeTests/5/console

Test check javaVendorVersion by `System.getProperty("java.vendor.version")` . 

#2544 add meta data to java.vendor.version for jdk11+. Should jdk8 have system property `java.vendor.version`?  Could someone please confirm this? If not we will need to limit tests to jdk11+. Otherwise this need to be fixed.

Similar system property `java.vendor.url.bug`, which isn't enabled for jdk8 with openj9:

15:01:28  FAILED: vmPropertiesPointToVendor
15:01:28  java.lang.AssertionError: expected [https://github.com/AdoptOpenJDK/openjdk-support/issues] but found [null]
15:01:28  	at org.testng.Assert.fail(Assert.java:96)
15:01:28  	at org.testng.Assert.failNotEquals(Assert.java:776)
15:01:28  	at org.testng.Assert.assertEqualsImpl(Assert.java:132)
15:01:28  	at org.testng.Assert.assertEquals(Assert.java:118)
15:01:28  	at org.testng.Assert.assertEquals(Assert.java:453)
15:01:28  	at org.testng.Assert.assertEquals(Assert.java:463)
15:01:28  	at net.adoptopenjdk.test.VendorPropertiesTest$AdoptOpenJDKPropertiesChecks.javaVendorUrlBug(VendorPropertiesTest.java:157)
15:01:28  	at net.adoptopenjdk.test.VendorPropertiesTest.vmPropertiesPointToVendor(VendorPropertiesTest.java:83)
15:01:28  	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
15:01:28  	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
```
https://ci.adoptopenjdk.net/job/build-scripts/job/jobs/job/jdk8u/job/jdk8u-linux-x64-openj9_SmokeTests/5/console

--- a/test/functional/buildAndPackage/src/net/adoptium/test/VendorPropertiesTest.java
+++ b/test/functional/buildAndPackage/src/net/adoptium/test/VendorPropertiesTest.java
@@ -106,7 +106,9 @@ public class VendorPropertiesTest {
         this.vendorChecks.javaVendor(System.getProperty("java.vendor"));
         this.vendorChecks.javaVendorUrl(System.getProperty("java.vendor.url"));
         this.vendorChecks.javaVendorUrlBug(System.getProperty("java.vendor.url.bug"));
-        this.vendorChecks.javaVendorVersion(System.getProperty("java.vendor.version"));
+        if (jdkVersion.isNewerOrEqual(10)) {
+            this.vendorChecks.javaVendorVersion(System.getProperty("java.vendor.version"));
+        }
         this.vendorChecks.javaVmVendor(System.getProperty("java.vm.vendor"));
         this.vendorChecks.javaVmVersion(System.getProperty("java.vm.version"));
     }

